<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(158, 202, 225);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="salesTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 8pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <column>
     <property name="text">
      <string>Order ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Retail Sales</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sales Date</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="exportPdfButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(255, 87, 87);</string>
    </property>
    <property name="text">
     <string>Print with Pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportExcelButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(255, 87, 87);</string>
    </property>
    <property name="text">
     <string>Print with Excel</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>621</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 8pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(255, 87, 87);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalPurchaseLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Price Purchase&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalSalesLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Retail Sales&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalIncomeLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Income&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
